storage/ops.go

```diff
@@ -398,11 +398,21 @@ func commonMakeFsCommand(bd *BlockDevice, args []string) ([]string, error) {
 }
 
 func commonMakePartCommand(bd *BlockDevice, start uint64, end uint64) (string, error) {
-	args := []string{
-		"mkpart",
-		bd.MountPoint,
-		fmt.Sprintf("%dM", start),
-		fmt.Sprintf("%dM", end),
+	var args []string
+	if end == 0 {
```

Comment:
```suggestion
     var trueEnd int64 = int64(end)

    if end == 0 {
        trueEnd = -1
    }

    args = []string{
        "mkpart",
        bd.MountPoint,
        fmt.Sprintf("%dM", start),
        fmt.Sprintf("%dM", trueEnd),
    }
```

Reply:
`var trueEnd int64 = int64(end)` gives a lint warning. So did it this way instead:
```
	var trueEnd int64
	if end == 0 {
		trueEnd = -1
	} else {
		trueEnd = int64(end)
	}
```